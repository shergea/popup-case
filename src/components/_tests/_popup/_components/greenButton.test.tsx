import React from "react";
import renderer from "react-test-renderer";
import GreenButton from "../../../popup/components/greenButton";

it("renders correctly", () => {
  const tree = renderer
    .create(<GreenButton text="Green Button Text" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
