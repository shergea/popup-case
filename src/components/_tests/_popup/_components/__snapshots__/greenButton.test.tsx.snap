// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="block w-full p-2.5 mt-5 text-base font-medium text-white buttonGreen"
  type="button"
>
  Green Button Text
</button>
`;
